packages/paste-website/src/components/ThemeSwitcher.tsx

```diff
@@ -1,13 +1,79 @@
 import * as React from 'react';
+import styled from '@emotion/styled';
+import {themeGet} from 'styled-system';
 import {ScreenReaderOnly} from '@twilio-paste/screen-reader-only';
+import {Box} from '@twilio-paste/box';
+import {Themes, ThemesType} from '../constants';
+import {useActiveSiteTheme} from '../context/ActiveSiteThemeContext';
 
 interface ThemeSwitcherProps {
   children?: React.ReactElement;
 }
+
+const StyledThemeSwitcherLabel = styled.label<{}>(props => ({
+  cursor: 'pointer',
+  display: 'inline-block',
+  padding: `${themeGet('space.space20')(props)} ${themeGet('space.space40')(props)}`,
+  '&:hover': {
+    textDecoration: 'underline',
+  },
+}));
+
+const StyledThemeSwitcherRadio = styled.input<{}>(props => ({
+  opacity: 0,
+  position: 'absolute',
+  [`&:focus + ${StyledThemeSwitcherLabel}`]: {
+    boxShadow: themeGet('shadows.shadowFocus')(props),
+    textDecoration: 'underline',
+  },
+  [`&:checked + ${StyledThemeSwitcherLabel}`]: {
+    backgroundColor: themeGet('backgroundColors.colorBackgroundPrimary')(props),
+    borderRadius: themeGet('radii.borderRadius10')(props),
+    color: themeGet('textColors.colorTextInverse')(props),
+  },
+}));
+
 export const ThemeSwitcher: React.FC<ThemeSwitcherProps> = () => {
+  const {theme, updateActiveSiteTheme} = useActiveSiteTheme();
+
+  const handleChange = (event: React.FormEvent<HTMLInputElement>): void => {
+    updateActiveSiteTheme(event.currentTarget.value as ThemesType);
+  };
+
   return (
-    <div>
-      <ScreenReaderOnly>Theme Switcher Placeholder</ScreenReaderOnly>
-    </div>
+    <Box
+      as="form"
+      borderColor="colorBorderLight"
+      borderRadius="borderRadius20"
+      borderStyle="solid"
+      borderWidth="borderWidth10"
+      padding="space10"
+    >
+      <Box as="fieldset" borderWidth="borderWidth0" padding="space0" margin="space0">
+        <legend>
```

Comment:
Could this be `<ScreenReaderOnly as="legend">Change the site theme</ScreenReaderOnly>`?  If not, should the legend tag be inside the `ScreenReaderOnly` element?

Reply:
Let me try the screenreaderonly as approach, that should do it. `legend` must be the first child of `fieldset` sop I can't wrap it.

Reply:
This worked, I just needed to add the `as` prop to `screenReaderOnly`